lib/tasks/mongoid_slug.rake

```diff
@@ -0,0 +1,16 @@
+namespace :mongoid_slug do
+  desc 'Goes though all documents and sets slug if not already set'
+  task set: :environment do |_, args|
+    ::Rails.application.eager_load! if defined?(Rails)
```

Comment:
@dblock if we add `::Rails.application.eager_load!` in Railtie then it will get execute unnecessarily whenever Rails console or server is started so I kept it here in rake task.